framework/src/main/java/org/tron/core/consensus/BlockHandleImpl.java

```diff
@@ -54,9 +59,12 @@ public BlockCapsule produce(Miner miner, long blockTime, long timeout) {
       manager.pushBlock(blockCapsule);
       tronNetService.broadcast(blockMessage);
     } catch (Exception e) {
+      metricsService.meterMark(MetricsService.NODE_STATUS, 1);
```

Comment:
There's something wrong with this metric， no need NODE_STATUS ， and NODE_STATUS is not a metric